src/tiptap-editor.vue

```diff
@@ -1,86 +1,99 @@
-                            ></path>
-                        </svg>
-                    </button>
-                </div>
-            </editor-menu-bar>
+                        <path
+                            fill="currentColor"
+                            d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
+                        ></path>
+                    </svg>
+                </button>
+            </div>
             <editor-content
                 :editor="editor"
                 :style="{ height: height }"
                 :id="id || null"
                 role="textbox"
                 class="editor__content"
                 aria-label="text area"
+                tabindex="-1"
             />
-        </div>
-        <div
-            v-if="maxCharacterCount"
-            :class="{ over: maxCharacterCountExceeded }"
-            class="character-count"
-            aria-live="polite"
-        >
-            {{ charactersRemaining }} characters remaining
+            <div class="character-count" v-if="maxCharacterCount && editor">
+                <svg
+                    height="20"
+                    width="20"
+                    viewBox="0 0 20 20"
+                    :class="
+                        maxCharacterCountExceeded
+                            ? 'character-count__graph--warning'
+                            : 'character-count__graph'
+                    "
+                >
+                    <circle r="10" cx="10" cy="10" fill="#e9ecef" />
+                    <circle
+                        r="5"
+                        cx="10"
+                        cy="10"
+                        fill="transparent"
+                        stroke="currentColor"
+                        stroke-width="10"
+                        :stroke-dasharray="`calc(${characterCountPercentage} * 31.4 / 100) 31.4`"
```

Comment:
it looks like for this to work in Safari & Firefox you need to provide a unit within `calc`. The suggestion below does fix it for firefox on my machine
```suggestion
                        :stroke-dasharray="`calc(${characterCountPercentage}px * 31.4 / 100) 31.4`"
```

Reply:
Good catch. Thanks!

Reply:
note that I definitely did not "just know" this :rofl:  I just forgot to link to the stack overflow question I found

https://stackoverflow.com/questions/65849051/calc-not-working-with-stroke-dashoffset-in-safari-and-firefox